web/src/integTest/java/com/microsoft/applicationinsights/azuretest/CompareTelemetryWithAzureExportTests.java

```diff
@@ -0,0 +1,342 @@
+package com.microsoft.applicationinsights.azuretest;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import com.microsoft.azure.storage.queue.CloudQueue;
+import com.microsoft.azure.storage.queue.CloudQueueClient;
+import com.microsoft.azure.storage.queue.CloudQueueMessage;
+import org.apache.commons.lang3.time.StopWatch;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import javax.xml.bind.JAXBException;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Created by moralt on 30/4/2015.
+ */
+public class CompareTelemetryWithAzureExportTests {
+    String runId = UUID.randomUUID().toString();
+    String configEnvironmentVariable = "INTEGRATION_TEST_CONFIGURATION";
+    IntegrationTestConfiguration config;
+    private static final int millisecondsInSecond = 1000;
+
+    /**
+     * Sends GET requests to server and expects to get similar result from app insights
+     * @throws Exception
```

Comment:
The result of GET is Http status code + body while from app insights export you're getting the logged telemetry, so I'm not sure that I understand what you mean by "expects to get similar result from app insights"...